src/Entity/Shipment.php

```diff
@@ -288,7 +287,7 @@ public function getShipFrom()
      *
      * @return Shipment
      */
-    public function setShipFrom(ShipFrom $shipFrom)
+    public function setShipFrom(ShipFrom $shipFrom = null)
```

Comment:
I think we can leave out the default declaration null in here, as the property automatically gets set to null at line 40. It can be useful for unsetting, but the better option is not to set it at all then I think. What do you think?